core/tests/protractor_desktop/topicsAndSkillsDashboard.js

```diff
@@ -94,17 +94,27 @@ describe('Topics and skills dashboard functionality', function() {
     await topicsAndSkillsDashboardPage.expectNumberOfTopicsToBe(2);
   });
 
-  it('should move published skill to unused skills section', async function() {
+  it('should move published skill to skills section', async function() {
```

Comment:
Can we adjust the name to reflect that this also tests deletion now?

Reply:
Yes, fixed this.
Done.